doc/devel/document.rst

```diff
@@ -1053,6 +1051,54 @@ subdirectory, but  :file:`galleries/users_explain/artists` has a mix of
 any ``*.rst`` files to a ``:toctree:``, either in the ``README.txt`` or in a
 manual ``index.rst``.
 
+Examples guidelines
+-------------------
+
+The gallery of examples contains visual demonstrations of matplotlib features. Gallery
+examples exist so that users can scan through visual examples. Unlike tutorials or user
+guides, gallery examples teach by demonstration, rather than by explanation or
+instruction.
+
+Gallery examples should contain a very brief description of *what* they demonstrate and,
+when relevant, *how* it is achieved. Avoid instructions or excessive explanation;
+instead, tag with related concepts and cross-link to relevant tutorials or user guides.
+
+Format
+^^^^^^
+
+All :ref:`examples-index` should aim to follow these guidelines:
+
+:Title: Describe content in a short sentence (approx. 1-6 words). Do not use *demo* as
+        this is implied by being an example. Avoid implied verbs such as "creating,
+        making, etc", e.g. "annotated heatmaps" is preferred to "creating annotated
+        heatmaps". Use the simple present tense when a verb is necessary, e.g. *Fill the
+        area between two curves*
+
+:Subtitle: In a short paragraph (approx 1-3 sentences) describe what visualization
+           technique is being demonstrated and how library features are used to execute
+           the technique, e.g. *Set bar color and bar label entries using the color and
+           label parameters of ~Axes.bar*
+
+:Plot: Clearly demonstrate the subject and, when possible, show edge cases and different
+       applications. While the plot should be visually appealing, prioritize keeping the
+       plot uncluttered.
+
+:Code: Write the minimum necessary to showcase the feature that is the focus of the
+       example. Avoid styling and annotation-such as setting titles, legends, colors,
+       etc.- when it will not improve the clarity of the example.
```

Comment:
```suggestion
       example. Avoid custom styling and annotations (titles, legends, colors, etc.)
       when it will not improve the clarity of the example.
```